components/demo/demo-snippet.js

```diff
@@ -87,29 +118,9 @@ class DemoSnippet extends LitElement {
 			.replace(/=""/g, '');           // replace empty strings for boolean attributes (="")
 	}
 
-	_handleDirChange() {
```

Comment:
Moved this into a more generic helper as I'm going to need the same code to apply `skeleton` to the stuff in the slot.